avrohugger-tools/src/test/scala/Util.scala

```diff
@@ -20,6 +20,20 @@ object Util {
     }
     readFile0(0)
   }
-  
+
+  class LineEndingAmbiguousMatcher(left: String) extends Matcher[String] {
+    def apply[S <: String](right: Expectable[S]): MatchResult[S] = {
+      val leftAsList = if (left.contains("\r\n")) left.split("\r\n") else left.split("\n")
+      val rightValue: String = right.value
+      val rightAsList = if (rightValue.contains("\r\n")) rightValue.split("\r\n") else rightValue.split("\n")
+
+      val res = leftAsList === rightAsList
+      result(res.isSuccess, res.message, res.message, right)
+    }
+  }
+
+  implicit class LineEndingAmbiguousMatcherString(s: String) extends ExpectationsCreation{
+    def ===/(other: String): MatchResult[String] = createExpectable(s).applyMatcher(new LineEndingAmbiguousMatcher(other))
```

Comment:
Test resource files contain LF endings and they are generated using system line endings. In case of windows CRLF, which breaks all the tests. This should compare files regardless of the line endings.